docs/book/src/developer/providers/v1.2-to-v1.3.md

```diff
@@ -47,7 +47,7 @@ The default value is 0, meaning that the volume can be detached without any time
 
 - The Kubernetes default registry has been changed from `k8s.gcr.io` to `registry.k8s.io`. Kubernetes image promotion currently publishes to both registries. Please
   consider publishing manifests which reference the controller images from the new registry (for reference [Cluster API PR](https://github.com/kubernetes-sigs/cluster-api/pull/7478)).
-- e2e tests are upgraded to use Ginkgo v2 (v2.2.0) and Gomega v1.20.1. Providers who use the test framework from this release will also need to upgrade, because Ginkgo v2 can't be imported alongside v1. Please see the [Ginkgo upgrade guide](https://onsi.github.io/ginkgo/MIGRATING_TO_V2), and note:
+- e2e tests are upgraded to use Ginkgo v2 (v2.4.0) and Gomega v1.22.1. Providers who use the test framework from this release will also need to upgrade, because Ginkgo v2 can't be imported alongside v1. Please see the [Ginkgo upgrade guide](https://onsi.github.io/ginkgo/MIGRATING_TO_V2), and note:
```

Comment:
Just to make sure, does this require any changes from providers? If so, it should go through our approval process (defined in https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/developer/release-cycle.md#release-cycle)

> All changes that impact providers' adoption of the new release should be announced in the provider updates section of the office hours meeting notes and approved in the PR or issue by both approvers and key affected providers

No objections from my end, just thought I'd point it out since the migration guide is being updated and we are now post beta.

Reply:
Good point and a good chance to refine further what "impact provider's adoption" means.

In my opinion changing dependencies minor/pach version should be fine even without approval as long as providers don't have to make code changes for that. If they would have to it should go through the approval process.

This means we would be fine with the fact that after providers are bumping to another beta and they are running `go mod tidy` some dependencies in their go.mod file change as well.


I looked through the release notes and they look good and non-disruptive to me.
That we didn't have to change anything in the test-framework or our e2e tests is another data point.
(gomega 1.23 would have made changes necessary > https://github.com/kubernetes-sigs/cluster-api/pull/7460#issuecomment-1302687929)

Reply:
Thank you for raising this important question Cécile. It's a great point. I also fully agree with Stefans excellent comment.